vmdb/spec/models/miq_product_feature_spec.rb

```diff
@@ -5,14 +5,14 @@
     it "empty table" do
       MiqRegion.seed
       MiqProductFeature.seed
-      MiqProductFeature.count.should eq(780)
+      MiqProductFeature.count.should eq(781)
```

Comment:
It looks like this was already 781 and should have been changed to 782.  Not sure why the bot and Github didn't complain about a merge conflict